pyqt5ac.py

```diff
@@ -143,6 +143,24 @@ def replaceVariables(variables_definition, string_with_variables):
     return string_with_variables
 
 
+def resolvePath(path: str, reference_path: str) -> str:
+    """
+    Translates relative paths into absolute paths, using the reference path as base.
+    Meaningful reference values for the caller might be the configuration file path, the script's path or the current
+    working directory.
+    :param path: path to resolve.
+    :param reference_path: path to be used as a reference to resolve absolute paths
+    :return: an absolute path corresponding to the relative path passed in input if it was relative, or the unchanged
+    input if it was an absolute path.
+    :raises: ValueError if the reference path is not absolute
+    """
+    if not os.path.isabs(path):
```

Comment:
rather
```
if os.path.isabs(path):
    return path
```
and then you can avoid the nested conditions. but ok I'll manicure this a little bit